pybv/io.py

```diff
@@ -246,43 +245,41 @@ def _optimize_channel_unit(resolution, unit):
 
 
 def _write_vhdr_file(vhdr_fname, vmrk_fname, eeg_fname, data, sfreq, ch_names,
-                     orientation='multiplexed', format='binary_float32',
-                     resolution=1e-7, unit='µV'):
+                     orientation, format, resolution, unit):
```

Comment:
I find it scary to have many default values for private functions with many parameters --> this can easily lead to uncaught errors.

So I remove all defaults except for the public write_raw_brainvision

Reply:
In addition, switching to keyword-only arguments can make total sense for functions with that many parameters!

Reply:
will do in a follow up pr